src/main/java/com/targomo/client/api/geo/DefaultSourceAddress.java

```diff
@@ -0,0 +1,78 @@
+package com.targomo.client.api.geo;
+
+import com.targomo.client.api.enums.TravelType;
+import com.targomo.client.api.pojo.LocationProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+/**
+ * AbstractClass for dealing with Source/Target H3 Addresses
+ */
+@MappedSuperclass @Getter @Setter
+public class DefaultSourceAddress extends AbstractLocation implements Location {
+
+	@Id
+	@Column(name = "identifier")
+	@GeneratedValue(strategy= GenerationType.TABLE)
+	private long identifier;
+
+	@Column(name = "travel_type")
+	private TravelType travelType;
```

Comment:
usually for `sources` we have something like:
```
"tm": {"car" {}}
```
why is it different here?
we need to support all travel type options : speed, frame etc...

Reply:
Sorry I wasn't clear, It's like that for the core but this is how the source would look like when using the travel options in other services like statistics

Reply:
Ok!